app/Mail/ContactFormCreated.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Website\Mail;
+
+use Blumilk\Website\Filament\Resources\ContactFormResource;
+use Blumilk\Website\Models\ContactForm;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class ContactFormCreated extends Mailable
+{
+    use Queueable;
+    use SerializesModels;
+
+    public function __construct(
+        public ContactForm $contactForm,
```

Comment:
I think we can make it protected?

```suggestion
        protected ContactForm $contactForm,
```